comms/storage/storage_main.go

```diff
@@ -38,13 +32,12 @@ func StorageMain() {
 		log.Fatal(err)
 	}
 
-	g := glue.New(GlobalNamespace, ReplicationFactor, jetstream.GetJetstreamContext())
-	e := web.NewServer(g)
+	ss := storageserver.NewProd(jetstream.GetJetstreamContext())
```

Comment:
fyi this global jetstream holder is removed here: https://github.com/AudiusProject/audius-protocol/pull/4683
can just use `peering.DialJetstream` one time and pass to storage server.

Reply:
thanks, I merged that in and confirmed that transcoding and image resizing are still working